lib/webpacker/compiler.rb

```diff
@@ -105,6 +105,7 @@ def webpack_env
       return env unless defined?(ActionController::Base)
 
       env.merge("WEBPACKER_ASSET_HOST"        => ENV.fetch("WEBPACKER_ASSET_HOST", ActionController::Base.helpers.compute_asset_host),
-                "WEBPACKER_RELATIVE_URL_ROOT" => ENV.fetch("WEBPACKER_RELATIVE_URL_ROOT", ActionController::Base.relative_url_root))
+                "WEBPACKER_RELATIVE_URL_ROOT" => ENV.fetch("WEBPACKER_RELATIVE_URL_ROOT", ActionController::Base.relative_url_root),
+                "WEBPACKER_CONFIG" => webpacker.config_path)
```

Comment:
JUST FYI...This causes a problem for me in testing: ```ActionView::Template::Error: no implicit conversion of Pathname into String
    /Users/jan/.rbenv/versions/2.7.1/lib/ruby/2.7.0/open3.rb:213:in `spawn'
    /Users/jan/.rbenv/versions/2.7.1/lib/ruby/2.7.0/open3.rb:213:in `popen_run'
    /Users/jan/.rbenv/versions/2.7.1/lib/ruby/2.7.0/open3.rb:101:in `popen3'
    /Users/jan/.rbenv/versions/2.7.1/lib/ruby/2.7.0/open3.rb:281:in `capture3'
    /Users/jan/.rbenv/versions/2.7.1/lib/ruby/gems/2.7.0/bundler/gems/webpacker-ce0133d1948d/lib/webpacker/compiler.rb:70:in `run_webpack'
    /Users/jan/.rbenv/versions/2.7.1/lib/ruby/gems/2.7.0/bundler/gems/webpacker-ce0133d1948d/lib/webpacker/compiler.rb:23:in `compile'``` ... adding `webpacker.config_path.to_s` fixes it.

Reply:
Could you please make a pr? thank you

Reply:
Added this PR https://github.com/rails/webpacker/pull/2573